demo/src/schemas/anomaly.ts

```diff
@@ -100,6 +100,7 @@ const schema: ChartSchemaElement[] = [
       x: 'created_date',
       y: 'failed_anomaly',
       size: 7,
+      dataComponent: <ExclamationCircleIcon />,
```

Comment:
@brumik How this should looklike?

Reply:
```ts
// In the schema:
dataComponent: 'exclamationCircleIcon'
...
// In the functions which we pass to the library on top level (not 100% sure in naming)
functions.customComponents = {
  excalamationCircleIcon: ExclamationCircleIcon // Pass it as a function without rendering
}
```

I believe this could work well. There may be needed a little fiddling how can we pass exactly down components, since it was quite few months I did this, but should be possible.